src/hand-tracking/CMakeLists.txt

```diff
@@ -10,11 +10,11 @@ find_package(OpenCV REQUIRED)
 
 find_package(YARP CONFIG REQUIRED
              COMPONENTS
-                 OS
```

Comment:
Most of the indentation corrected in this commit were actually "correct", meaning that they were correct for me eheheh 😆 The comment I made about the "wrong indentation" was just about a couple of places. I would delete this commit and just change the line I told you in the previous round of comments. Not a big deal, I'm also open to change this in some next commit if you prefer not to spend more time on this.

Reply:
For me it is ok to remove the commit and just change the few lines you told me. 

However it is unclear to me whether I have to change the line containing `utils.{h/cpp}` (that for me are correctly indented) or those of `Walkman` that seem to me the only not indented with all the others.

Reply:
Do you have links to the changes. Navigating through the commits is getting complicated ahahahah!

Reply:
Yep, the commit where the `CMakeLists.txt` was changed is [this](https://github.com/robotology/visual-tracking-control/pull/3/commits/e19652fb49ccdf3af2af082d5f997d3bb4436daf).

Reply:
Have a look at [this](https://github.com/robotology/visual-tracking-control/blob/e19652fb49ccdf3af2af082d5f997d3bb4436daf/src/hand-tracking/CMakeLists.txt) file. You can see that some lines are not properly indented. In particular
  1. [CMakeLists.txt#L130](https://github.com/robotology/visual-tracking-control/blob/e19652fb49ccdf3af2af082d5f997d3bb4436daf/src/hand-tracking/CMakeLists.txt#L130)
  2. [CMakeLists.txt#L154-L155](https://github.com/robotology/visual-tracking-control/blob/e19652fb49ccdf3af2af082d5f997d3bb4436daf/src/hand-tracking/CMakeLists.txt#L154-L155)
  3. [CMakeLists.txt#L154-L155](https://github.com/robotology/visual-tracking-control/blob/e19652fb49ccdf3af2af082d5f997d3bb4436daf/src/hand-tracking/CMakeLists.txt#L154-L155)

Reply:
Ok, now we agree. I will delete the commit as you said and change only those lines.

Reply:
Cool thanks!